### ISSUE TITLE
Typo in notification of renamed builder

### ISSUE THREAD

**Issue by Faxix** [ldtteam/minecolonies#7197]
TL:DR:
Builder with custom name (named „Builder 2“ via Name tag) have „typo“ in notification about starting work – missing „: “ between name and rest of message „I’ve started work…“.

### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version :0.14.97-ALPHA 
- Structurize Version: 0.13.163-ALPHA
- Related Mods and their Versions: (from Direwolf20 1.16 pack v1.19.0)

## Expected behavior

Message with ": " -> Builder Default-citizen-name: I’ve started work...

## Actual behavior

Message without ": " -> Builder Custom-citizen-nameI’ve started work...

## Steps to reproduce the issue

1. Use anvil renamed Name tag on builder
2. Start work with builder

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.
